src/lua/luaCommands.c

```diff
@@ -39,14 +40,13 @@ void ExecLuaInterpreter(struct Serial *serial, unsigned int argc, char **argv)
                      "to leave");
         put_crlf(serial);
 
-        cmd_context *cmdContext = get_command_context();
-        char *luaLine = cmdContext->lineBuffer;
+        static char luaLine[256];
```

Comment:
Yep.  I have a ticket out to be able to pass in the object with the buffer to handle this.. just not part of this patch and not a requirement for release.  Hence this hack.